.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,11 @@
+# Info
+
+* Please don't force push to PRs, it removes the history, creates bad notifications, and we will squash and merge in the end.
+* Feel free to ping for a review after a few days (@simondanisch, @ffreyer). We can't guarantee a review in a certain time frame, but we can guarantee to forget about PRs after a while.
+* The easier a PR is to review, the easier we can merge it:
+    * The more tests you add, the easier it is to see that your change works
+    * The clearer the problem being solved the easier.
+
```

Comment:
Maybe this could be hidden behind an HTML comment? Similar to the JupyterLab pull request template: https://raw.githubusercontent.com/jupyterlab/jupyterlab/main/.github/pull_request_template.md

Reply:
It'll be visible to all, and won't be rendered if the user doesn't delete those lines.

Reply:
```suggestion
<!--
* Please don't force push to PRs, it removes the history, creates bad notifications, and we will squash and merge in the end.
* Feel free to ping for a review after a few days (@simondanisch, @ffreyer). We can't guarantee a review in a certain time frame, but we can guarantee to forget about PRs after a while.
* The easier a PR is to review, the easier we can merge it:
    * The more tests you add, the easier it is to see that your change works
    * The clearer the problem being solved the easier.
-->